www/includes/easyparliament/templates/html/section/day.php

```diff
@@ -0,0 +1,89 @@
+<div class="full-page__row">
+
+    <div class="business-section">
+        <div class="business-section__header">
+            <h1 class="business-section__header__title">
+                <?= $parent_title ?>
+            </h1>
+            <p class="business-section__header__date">
+                <?= format_date( $info['date'], LONGERDATEFORMAT ) ?>
+            </p>
+            <div class="business-section__header__description">
+                <?php include '_' . $section . '_desc.php'; ?>
```

Comment:
Even at quite a large browser size (I'm currently on 1471x800), half of this page is heading; I see 3 entries on e.g. `/debates/?d=2013-07-10` as opposed to 10 or so in the old design. Could the description be relegated to the side where it used to be?